alexandria/core/management/commands/generate_missing_thumbnails.py

```diff
@@ -0,0 +1,21 @@
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from psutil import virtual_memory
+from tqdm import tqdm
+
+from alexandria.core.models import File
+from alexandria.core.utils import create_thumbnail
+
+
+class Command(BaseCommand):
+    help = "Finds documents that don't have a thumbnail and generates them."
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        for file in tqdm(
+            File.objects.filter(variant="original", renderings__isnull=True)
+        ):
+            create_thumbnail(file)
+            if virtual_memory().available < 300000000:
+                print("about to run out of memory, stopping")
+                break
```

Comment:
Will it just fill up memory otherwise?

Reply:
Yeah, similar to what we saw with the encryption command